src/httpserver/core.clj

```diff
@@ -10,6 +11,10 @@
 (defn -main [& args]
   (let [{:keys [options]} (arguments/build-map args)]
     (let [server (HttpServer. (:port options) (:directory options))
-          teapot-handler (teapot/new-teapot-handler)]
-      (.addRoute server "GET" "/teapot" teapot-handler)
+          teapot-handler (teapot/new-teapot-handler)
+          ok-handler (ok/new-ok-handler)
+          ]
```

Comment:
Just a style thing—the closing `]` for `let` bindings is typically on the same line as the final binding. Also, no line empty line break between the bindings and the next forms (line below this)